modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -3791,4 +3791,44 @@
       end
     end
   end
+
+  describe 'POST #submit_form_526 not using md5 lookup' do
+    let(:anticipated_separation_date) { 2.days.from_now.strftime('%Y-%m-%d') }
+    let(:active_duty_end_date) { 2.days.from_now.strftime('%Y-%m-%d') }
+    let(:data) do
+      temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans', 'disability_compensation',
+                             'form_526_json_api.json').read
+      temp = JSON.parse(temp)
+      attributes = temp['data']['attributes']
+      attributes['serviceInformation']['federalActivation']['anticipatedSeparationDate'] = anticipated_separation_date
+      attributes['serviceInformation']['servicePeriods'][-1]['activeDutyEndDate'] = active_duty_end_date
+
+      temp.to_json
+    end
+    let(:schema) { Rails.root.join('modules', 'claims_api', 'config', 'schemas', 'v2', '526.json').read }
+    let(:veteran_id) { '1013062086V794840' }
+    let(:submit_path) { "/services/claims/v2/veterans/#{veteran_id}/526" }
+
+    it 'creates a new claim if duplicate submit occurs (does not use md5 lookup)' do
+      mock_ccg(scopes) do |auth_header|
+        VCR.use_cassette('claims_api/disability_comp') do
```

Comment:
Since this is running twice inside the same test for two submissions I needed to use the cassette like this instead of around the whole thing like it is used above.